### DISCUSSION TITLE
12-04

### DISCUSSION THREAD

**Discussion by hanseulhee** [ChristmasTest/christmas#40] [Daily_Meeting]
### ⛄️ 오늘의 TMI
수영님 : 음.... 오늘은 **등 + 이두** 하는날 (등두) 헬스 영업중. 다짜고짜 물어보셈 ㅋㅅㅋ
수님 :  오늘부터 헬스를 시작한다!! 남자친구랑 약속 워너비 몸매를 위해 헬짱의 길을...
슬희님 : 4시 PT
윤혁님 : 회의 끝나고 운동가기
<br>

### ⛄️ 내용 공유
JEST 공부 했나요?!
모두 안함!
<br>

### ❄️ 오늘 할일
⛄️ 밥 먹기
⛄️ 강의 보기
⛄️ 운동 가기
<br>

### ⛄️ TMI
- 슬희님 제 차례인데 왜 뺏으신 거죠?

**Comment by hanseulhee** [comment]
쏘리 ㅋ

**Comment by ngsy** [comment]
수님 운동 화이팅👍  헬'수'

**Reply by hanseulhee** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by miniposi** [reply]
아놔 ㅋ
